src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -35,7 +36,8 @@
   public static final String MAX_RETRIES_CONFIG = "max.retries";
   public static final String TYPE_NAME_CONFIG = "type.name";
   public static final String TOPIC_INDEX_MAP_CONFIG = "topic.index.map";
-  public static final String RETRY_BACKOFF_MS_CONFIG = "retry.backoff.ms";
+  public static final String MIN_RETRY_BACKOFF_MS_CONFIG = "retry.backoff.ms";
+  public static final String MAX_RETRY_BACKOFF_MS_CONFIG = "max.retry.backoff.ms";
```

Comment:
I feel that we might not strike the best compromise by introducing another config property and also not renaming the old one (wouldn't be great the break compatibility of course). 

Is `max.retry.backoff.ms` all that necessary? The users will have to figure out that this is a global max (including the backoffs with the given number of retries). And maybe that's not obvious or it becomes a bit tedious to get it right. 

Why not stay with the maximum that is implied by the `retry.backoff.ms`, `max.retries` and the fact that the backoff is exponential (at a power of 2)? Also max can't have an adjustable default (relative to what min will be). Thus it's destined to require adjustment every time someone changes min. 

I feel we don't gain much by introducing it. Just a thought